resources/public/api/conf/2.0/examples/property_period_summaries_list/response.json

```diff
@@ -2,13 +2,13 @@
   "submissions": [
     {
       "submissionId": "4557ecb5-fd32-48cc-81f5-e6acd1099f3c",
-      "fromDate": "2021-08-18",
-      "toDate": "2021-09-18"
+      "fromDate": "2021-04-06",
+      "toDate": "2021-07-05"
     },
     {
       "submissionId": "46954a50-5127-41ec-bf63-0242ac130002",
-      "fromDate": "2021-06-18",
-      "toDate": "2021-07-18"
+      "fromDate": "2021-07-06",
+      "toDate": "2021-10-04"
```

Comment:
toDate value should be "2021-10-05" ? as per [this guidance page ](https://www.gov.uk/guidance/using-making-tax-digital-for-income-tax#:~:text=in%20this%20table.-,Quarterly%20period,5%20May,-Use%20calendar%20quarterly)